frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -1,5 +1,28 @@
 import * as _ from 'lodash';
 import { Alert } from '@console/internal/components/monitoring';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephStorageLabel = 'cluster.ocs.openshift.io/openshift-storage';
 
 export const filterCephAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'ceph');
+
+export const getFilteredPVCsData = (pvcsData: K8sResourceKind[]): K8sResourceKind[] =>
+  pvcsData.filter((pvc: K8sResourceKind) =>
+    cephStorageProvisioners.includes(
+      _.get(pvc, 'metadata.annotations["volume.beta.kubernetes.io/storage-provisioner"]'),
+    ),
+  );
+
+export const getFilteredPVsData = (pvsData: K8sResourceKind[]): K8sResourceKind[] =>
+  pvsData.filter((pv: K8sResourceKind) =>
+    cephStorageProvisioners.includes(
+      _.get(pv, 'metadata.annotations["pv.kubernetes.io/provisioned-by"]'),
+    ),
+  );
+
+export const getFilteredNodesData = (NodesData: K8sResourceKind[]): K8sResourceKind[] =>
+  NodesData.filter((node: K8sResourceKind) =>
```

Comment:
nit: you dont need to specify again that node is `K8sResourceKind` - it is derived by TS automatically (applies to all `getFiltered*` functions)